lib/presentation/home/widgets/current_upcoming_ca_layout.dart

```diff
@@ -38,117 +38,148 @@ class CurrentAndUpcomingLayout extends StatelessWidget {
-                                        borderRadius:
-                                            BorderRadius.circular(20))),
-                                backgroundColor:
-                                    MaterialStateProperty.all(kAccentColor)),
-                            onPressed: () {},
-                            icon: const Icon(
-                              Icons.check,
-                              color: Colors.white,
+                  // const SizedBox(height: 3),
+                ],
+              ),
+            ),
+            ...crowdActionsList.map(
+              (e) => Container(
+                height: 148,
+                width: MediaQuery.of(context).size.width,
+                margin: const EdgeInsets.symmetric(vertical: 10),
+                child: Card(
+                  shape: RoundedRectangleBorder(
+                      borderRadius: BorderRadius.circular(20)),
+                  elevation: 4,
+                  child: Row(
+                    children: [
+                      Container(
+                        decoration: BoxDecoration(
+                            color: Colors.black,
+                            borderRadius: BorderRadius.circular(15),
+                            image: DecorationImage(
+                                image:
+                                    AssetImage(e['crowdaction_img'].toString()),
+                                fit: BoxFit.cover)),
+                        margin: const EdgeInsets.only(left: 10),
+                        height: 128,
+                        width: 100,
+                      ),
+                      Expanded(
+                        child: Column(
+                          crossAxisAlignment: CrossAxisAlignment.start,
+                          children: [
+                            CustomLabelForCrowdAction().customLabelBox(
+                                const Icon(
+                                  Icons.check,
+                                  size: 20,
+                                  color: kSecondaryColor,
+                                ),
+                                const Text(
+                                  'Sign up now',
+                                  style: TextStyle(
+                                    fontSize: 12,
+                                    fontWeight: FontWeight.w700,
+                                    color: kSecondaryColor,
+                                  ),
+                                ),
+                                kAccentColor)!,
+                            // Container(
+                            //     padding: const EdgeInsets.only(
+                            //         top: 10, left: 10, right: 10),
+                            //     decoration: BoxDecoration(
+                            //         borderRadius: BorderRadius.circular(16)),
+                            //     child: const Text('Sign up now')),
```

Comment:
Delete comment

Reply:
I've deleted this file as Mathias has made a copy of it with appropriate function names.